app/Welcome.tsx

```diff
@@ -4,15 +4,24 @@ import { Button } from "react-bootstrap";
 export function Welcome(props: { onLogout: () => void }): ReactElement {
   const { onLogout } = props;
   return (
-    <div style={{ display: "flex" }}>
-      SUCCESS
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        color: "#FFF",
+        fontSize: "20px",
+        alignItems: "center",
+      }}
+    >
```

Comment:
do you think most pages will have this similar type of styling? let's not mix the styles of the page container and the text, ie wrap the text in a different div and style that separately

Reply:
put a styled const in utils 'page header styling' to be reused across pages 
https://github.com/auberginearms/starterApp/commit/2b58d380aa4d7f717ff9ff479827a7ce551fc7eb